### ISSUE TITLE
improve flakey secure compose recipients

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-browser#1505]
This shows up quite a bit in automated tests, and sometimes one can experience it in real life too.

```
Page 3 (active) chrome-extension://ilaeoeclfgdgbbjpghpcmnlhmkfccdij/chrome/elements/compose.htm?account_email=flowcrypt.compatibility%40gmail.com&parent_tab_id=0&frameId=none&
(no console messages)
ACTIVE alert: Could not send message: Error: Please re-enter recipients marked in red color.
```

![image](https://user-images.githubusercontent.com/6306961/51684330-b4a10700-1fe3-11e9-883b-bc1748ab9d3e.png)

**Comment by tomholub** [comment]
```
Page 1 (active) chrome-extension://eaafkghcgcecpbpcbcdmfhgkaaaalbkn/chrome/elements/compose.htm?frameId=frame_ptdwxQUvkA&isReplyBox=___cu_false___&placement=gmail&acctEmail=flowcrypt.test.trial%40gmail.com&parentTabId=12%3A0

(no console messages)

confirm: The files are over 5 MB. Advanced users can send files up to 25 MB. Try it free for 30 days.
ACTIVE alert: Could not send message: Error: Please re-enter recipients marked in red color.
```
![image](https://user-images.githubusercontent.com/6306961/51702253-3bb5a580-200b-11e9-9182-3f914a53ebc0.png)

**Comment by tomholub** [comment]
mostly fixed by #1542
